README.md

```diff
@@ -4,21 +4,26 @@
 
 # [Element Finance](https://element.fi) market simulation and analysis
 
-This project is a work-in-progress. All code is provided as is and without guarantee.
+This project is a work-in-progress. All code is provided as is and without guarantee. Documentation can be found [here](elf-simulations.vercel.app).
```

Comment:
we should really update DNS with elf-simulations.element.fi or something

Reply:
that would be dope